applications/adv_networking_bench/cpp/dpdk_bench_op_rx.h

```diff
@@ -95,144 +93,215 @@ class AdvNetworkingBenchDefaultRxOp : public Operator {
                          42);
   }
 
-  void free_bufs() {
-    while (out_q.size() > 0) {
-      const auto first = out_q.front();
-      if (cudaEventQuery(first.evt) == cudaSuccess) {
-        for (auto m = 0; m < first.num_batches; m++) {
-          adv_net_free_all_pkts_and_burst(first.msg[m]);
+  // Free buffers if CUDA processing/copy is complete
```

Comment:
What does `CUDA processing/copy` encapsulate here? DPDK packet re-ordering is done?

Reply:
reordering above for GPU packets, copy from CPU above for CPU packets

lmk how you think that comment can be improved if that wasn't clear

Reply:
Could you please add a comment linking to the existing processing summary in the README doc? https://github.com/nvidia-holoscan/holohub/tree/883997d168604c023c2445a72bcb08ef13923645/applications/adv_networking_bench/cpp#receiver

Reply:
that documentation is outdated and incomplete/wrong, will stick to just changes in this current file, docs refactoring is next.